contracts/staking/interfaces/IValidatorManager.sol

```diff
@@ -41,7 +41,7 @@ struct ValidatorManagerSettings {
 struct SubnetConversionData {
     bytes32 convertSubnetTxID;
     bytes32 blockchainID;
-    bytes validatorManagerAddress;
+    address validatorManagerAddress;
```

Comment:
Should we leave this as bytes as that's what spec defines it as in terms of encoding? It's not fixed 20 bytes but 4 bytes for len + `len(validationManagerAddress)` That would make any automated packing/unpacking easier and then can convert to address at usage time.